net-snmp/run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+(
+    . /etc/sysconfig/snmpd
+    if test "$WAIT_FOR_CONFIGURATION" == 1; then
+        systemd-notify --ready
+        exec sleep 9999d
```

Comment:
yes, it must be created interactively since you need to specify user/password (and can create multiple users).  The configuration should usually be between the `atomic install --system` and the `systemctl start $NAME`, just in this case it is more difficult since it requires tools that are inside the container image and at the same time we cannot run the daemon.